### ISSUE TITLE
Cannot import name 'image_preprocessing'

### ISSUE THREAD

**Issue by trystbinx** [tensorflow/tensorflow#43147]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): Source
- TensorFlow version (use command below): 2.1.0
- Python version: 3.6.12
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.1/7.6
- GPU model and memory:

**Describe the current behavior**
I am currently struggling to proceed due to having an unexpected error message containing 'ImportError: cannot import name 'image_preprocessing''.

I am doing a project related to Object Detection (credits to: EdjeElectronics). Upon installing Tensorflow, Keras, CUDA, cudnn, Visual Studio Microsoft Redistribution 2019, I tried to do testing 'python ~\object_detection\builders\model_builder_tf2_test.py' to see if everything is correctly installed. However, I keep receiving this error.

**Describe the expected behavior**
```
(tensor-gpu) PS C:\Users\User\tensorflow11\models\research> python object_detection\builders\model_builder_tf2_test.py
2020-09-11 12:40:46.813673: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
Traceback (most recent call last):
  File "object_detection\builders\model_builder_tf2_test.py", line 24, in <module>
    from object_detection.builders import model_builder
  File "C:\Users\User\tensorflow11\models\research\object_detection\builders\model_builder.py", line 65, in <module>
    from object_detection.models import ssd_efficientnet_bifpn_feature_extractor as ssd_efficientnet_bifpn
  File "C:\Users\User\tensorflow11\models\research\object_detection\models\ssd_efficientnet_bifpn_feature_extractor.py", line 33, in <module>
    from official.vision.image_classification.efficientnet import efficientnet_model
  File "C:\Users\User\tensorflow11\models\official\vision\image_classification\efficientnet\efficientnet_model.py", line 37, in <module>
    from official.vision.image_classification import preprocessing
  File "C:\Users\User\tensorflow11\models\official\vision\image_classification\preprocessing.py", line 25, in <module>
    from official.vision.image_classification import augment
  File "C:\Users\User\tensorflow11\models\official\vision\image_classification\augment.py", line 31, in <module>
    from tensorflow.python.keras.layers.preprocessing import image_preprocessing as image_ops
ImportError: cannot import name 'image_preprocessing'
```

Appreciate if experienced TF developers to state where/what I am doing wrong.

Thank you.

**Comment by ravikyram** [comment]
@trystbinx 

Import of image_ops is unfortunately only available with TF 2.2+ .

Please, upgrade to TF 2.3 and check if the issue still persists. Thanks!

**Comment by trystbinx** [comment]
> 
> 
> @trystbinx
> 
> Import of image_ops is unfortunately only available with TF 2.2+ .
> 
> Please, upgrade to TF 2.3 and check if the issue still persists. Thanks!

Thank you for your reply. Yes, the issues still persists, but a different error output.

TF version is 2.2.0
Keras is 2.3.1

Installed CUDA 10.1, cudnn, 7.6.5, set path for both.

I am not sure what I am doing wrong. I have been looking for answers on the stackoverflow/github, but has not resolved this issue since a few days ago.

Is there anything I should do to resolve this issue?

```
(toogpu) PS C:\Users\User\tensorflow11\models\research> python
Python 3.6.12 |Anaconda, Inc.| (default, Sep  9 2020, 00:29:25) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import keras
Using TensorFlow backend.
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: A dynamic link library (DLL) initialization routine failed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\__init__.py", line 3, in <module>
    from . import utils
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\utils\__init__.py", line 6, in <module>
    from . import conv_utils
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\utils\conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\backend\__init__.py", line 1, in <module>
    from .load_backend import epsilon
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\backend\load_backend.py", line 90, in <module>
    from .tensorflow_backend import *
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\keras\backend\tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\__init__.py", line 41, in <module>
    from tensorflow.python.tools import module_util as _module_util
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\__init__.py", line 50, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 69, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "C:\ProgramData\Anaconda3\envs\toogpu\lib\imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: A dynamic link library (DLL) initialization routine failed.
```

**Comment by ravikyram** [comment]
@trystbinx 

This is Tensorflow installation error.
Can you try it in fresh virtual environment and try to install. It is mostly due to compatibility issue.

What is make/model of your cpu?
I suspect your cpu model does not support AVX instructions sets.See [hardware requirements](https://www.tensorflow.org/install/pip?lang=python3#hardware-requirements)
Make sure to download [the latest microsoft visual c++ redistributable from here](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
.Also, please follow the instructions from to install from [Tensorflow website](https://www.tensorflow.org/install/source_windows).

Thanks!
